<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoPostGISLinker</class>
 <widget class="QWidget" name="GeoPostGISLinker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PostGIS Linker</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Adobe Arabic</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="Username_lineEdit">
        <property name="text">
         <string>whu_gis4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Adobe Arabic</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="Password_lineEdit">
        <property name="text">
         <string>whugis4WYKnb</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Adobe Arabic</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Server Host:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Adobe Arabic</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Database:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="Database_lineEdit">
        <property name="text">
         <string>SHP</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QPushButton" name="Link_Button">
        <property name="text">
         <string>Link PostGIS</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QListView" name="Tables_listView"/>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="Add_Button">
        <property name="text">
         <string>Add to Map</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="Cancel_Button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Hostname_lineEdit">
        <property name="text">
         <string>pgm-bp124o1lka7h382fyo.pg.rds.aliyuncs.com</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="Port_lineEdit">
        <property name="text">
         <string>1433</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
